src/Mod/Start/Gui/DlgStartPreferences.ui

```diff
@@ -111,7 +111,7 @@ By using &quot;;;&quot; to separate paths, you can add several folders here.</st
       <item row="0" column="1">
        <widget class="Gui::PrefCheckBox" name="checkBoxCloseAfterLoading">
         <property name="toolTip">
-         <string>If checked, will automatically close the Start page when FreeCAD launches</string>
+         <string>Closes the start page automatically when FreeCAD launches</string>
```

Comment:
"Closes" seems odd for a checkbox -- checking the box doesn't close anything. Maybe "Close" in this case?